java/clients/src/main/java/sleeper/status/report/filestatus/StandardFileStatusReporter.java

```diff
@@ -41,25 +52,42 @@ public void report(FileStatus fileStatusReport, boolean verbose) {
             printFileInfoList("Ready_to_be_garbage_collected", fileStatusReport.getGcFiles());
             printFileInfoList("Active", fileStatusReport.getActiveFiles());
         }
-        System.out.println("Total number of records in all active files = " + fileStatusReport.getTotalRecords());
-        System.out.println("Total number of records in leaf partitions = " + fileStatusReport.getTotalRecordsInLeafPartitions());
-        System.out.println("Percentage of records in leaf partitions = " + (fileStatusReport.getTotalRecordsInLeafPartitions() / (double) fileStatusReport.getTotalRecords()) * 100.0);
+        out.println("Total number of records in all active files = " + abbreviatedRecordCount(fileStatusReport.getTotalRecords()));
+        out.println("Total number of records in leaf partitions = " + abbreviatedRecordCount(fileStatusReport.getTotalRecordsInLeafPartitions()));
+        out.println("Percentage of records in leaf partitions = " + (fileStatusReport.getTotalRecordsInLeafPartitions() / (double) fileStatusReport.getTotalRecords()) * 100.0);
 
     }
 
     private void printPartitionStats(FileStatus.PartitionStats partitions, String type) {
         if (partitions.getTotal() > 0) {
-            System.out.println("Number of files in " + type + " partitions:" +
+            out.println("Number of files in " + type + " partitions:" +
                     " min = " + partitions.getMinSize() +
                     ", max = " + partitions.getMaxMax() +
                     ", average = " + partitions.getAverageSize());
         } else {
-            System.out.println("No files in " + type + " partitions");
+            out.println("No files in " + type + " partitions");
         }
     }
 
-    private static void printFileInfoList(String type, List<FileInfo> strings) {
-        System.out.println(type + ":");
-        strings.forEach(System.out::println);
+    private void printFileInfoList(String type, List<FileInfo> strings) {
+        out.println(type + ":");
+        strings.forEach(out::println);
     }
+
+    private static final long K_COUNT = 1000;
+    private static final long M_COUNT = 1000000;
+    private static final long G_COUNT = 1000000000;
```

Comment:
It might be more readable to write these using _, e.g. 1_000, 1_000_000

Reply:
Done